RecoHGCal/TICL/plugins/TrackstersMergeProducer.cc

```diff
@@ -390,45 +389,49 @@ void TrackstersMergeProducer::produce(edm::Event &evt, const edm::EventSetup &es
           for (auto otherTracksterIdx : trackstersTRKEMwithSameSeed) {
             auto thisPt = tracksterTotalRawPt + trackstersMergedHandle->at(otherTracksterIdx).raw_pt() - t.raw_pt();
             closestTrackster = std::abs(thisPt - track.pt()) < minPtDiff ? otherTracksterIdx : closestTrackster;
-            usedTrackstersMerged[otherTracksterIdx] = true;
           }
           tracksterTotalRawPt += trackstersMergedHandle->at(closestTrackster).raw_pt() - t.raw_pt();
+          usedTrackstersMerged[closestTrackster] = true;
```

Comment:
Is this a bug fix? Should we expect any effect on the results (i.e., should you update the validation plots?)

Reply:
yes, the results in PF validation should change now, we are rerunning it